system/jlib/jstats.cpp

```diff
@@ -377,7 +377,14 @@ static StringBuffer & formatLoad(StringBuffer & out, unsigned __int64 value)
 static StringBuffer & formatSkew(StringBuffer & out, unsigned __int64 value)
 {
     //Skew stored as 10000 = perfect, display as percentage
-    return out.appendf("%.2f%%", ((double)(__int64)value) / 100.0);
+    const __int64 & sval = (__int64) value;
```

Comment:
@shamser Why the reference?  That looks like a bug to me - assigning a temporary value to a reference.

Reply:
I'm using it as a quick way to cast from unsigned to signed.  I need the signed for both percent calculation and for comparing the range.  I can do it without using reference but there'll be 2 copies of the value.

Reply:
because of the if/else >= , <= would calling:
```
unsigned __int64 aval := labs((__int64)value);
```
then you only need e.g. if (aval >= 10000)

Reply:
if you are doing that, then you need to use std::abs(x).  long may be smaller than __int64.

Reply:
I could take the absolute value for the comparison but I need the -ve value for display.